website/docs/reference/events-logging.md

```diff
@@ -65,6 +67,7 @@ For events where it's available, `node_info` will include:
 | `node_finished_at` | Timestamp when node processing completed |
 | `node_name` | Name of this model/seed/test/etc |
 | `node_path` | File path to where this resource is defined |
+| `node_relation` | <VersionBlock firstVersion="1.5">Nested object containing this node's database representation: `database`, `schema`, `alias`, and full `relation_name` with quoting & inclusion policies applied</VersionBlock><VersionBlock lastVersion="1.4">Added in [v1.5](/reference/events-logging?version=1.5#node_info-fields)</VersionBlock> |
```

Comment:
This is me being a little bit fancy with `VersionBlock`

<img width="859" alt="Screenshot 2023-04-12 at 21 45 07" src="https://user-images.githubusercontent.com/13897643/231568218-08a90cd4-0e4f-4a18-bd11-18867639acd1.png">

<img width="848" alt="Screenshot 2023-04-12 at 21 44 57" src="https://user-images.githubusercontent.com/13897643/231568212-6dde9510-a404-463c-b815-2405012cbd28.png">

Reply:
🤩

FFFTW! (Fancy Footwork For The Win)

Reply:
This is a fantastic idea, but it doesn't function as expected due to technical limitations. We've tried this in the past, and the behavior here is the same - it links to the page properly but doesn't change the version of the docs when written as a markdown link. It works if you put the full `HTTP` link, but the downside there is it opens in a new tab (this is expected behavior in the current Docusaurus implementation).

Reply:
Ah - alas. I'll remove the link, but leave in the conditional table contents